### PR TITLE
Task3 (DOM Manipulation)

### PR DESCRIPTION
**Pull request by mishalfirdous**

### CODE REVIEW

File: style.css
Comment (abdullahshahzad7890) [inline style.css:1]: why do we have two separate files for CSS?

Comment (mishalfirdous) [reply style.css:1]: one file for index and potfolio and one for singup page

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="stylesheet.css" />
+    <header>
+      <div class="login">
+        <ul>
+          <li><a href="index.html">index</a></li>
```
Comment (abdullahshahzad7890) [inline portfolio.html]: what is index?

File: style.css
Comment (abdullahshahzad7890) [reply style.css:1]: why so?

File: portfolio.html (hunk 1 above)
Comment (mishalfirdous) [reply portfolio.html]: fixed.

File: style.css
Comment (mishalfirdous) [reply style.css:1]: merge all the styles into one sheet, and resolve!

**Review by abdullahshahzad7890 — APPROVED** [review]

**Pull request merged**